pages/docs/contributing/maintainers.mdx

```diff
@@ -0,0 +1,87 @@
+import DocsLayout from "../../../components/layouts/DocsLayout.js";
+
+## Maintainers
+
+- Only assign issues with the label `status: ready for dev`
+- Each person should only be assigned 1 open issue at a time
+
+---
+
+> `avatar` json property is included
+
+This is no longer used, please leave an inline comment with the code sugestion to remove it - this will allow another maintainer to accept the changes and we can get the Pull Request merged without delay.
+
+```diff
+- "avatar": "http://github.com/SaraJaoude.png",
++
+```
+
+If you wish to make the fix yourself, you can merge to a temporary branch and fix, then create a new Pull Request.
+
+_Note: multiple Pull Requests that need fixing can go into this temporary branch, but please make sure this branch is not open for longer than a few hours to avoid any confusion._
```

Comment:
This is a great start @eddiejaoude! 😄 However, the word "this" in "please make sure this branch" implies that there is a specific branch in place, so maybe we can say this instead:
Note: multiple Pull Requests that need fixing can go into this temporary branch, but please make sure the branch is not open for longer than a few hours to avoid any confusion.

Reply:
Thank you Chrissy! Btw you can click the "suggest" button, which will add the changes in the inline comment and you can amend it there with your suggestions, there are 2 reasons why this is great...
1. I can just click accept 
2. you get a green screen for making the changes/commit

<img width="822" alt="Screenshot 2023-03-07 at 11 54 05" src="https://user-images.githubusercontent.com/624760/223324232-62cb33d1-819a-4564-aacd-05a510f84527.png">